libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -222,57 +212,43 @@ class AC_AttitudeControl {
     // Proportional controller with piecewise sqrt sections to constrain second derivative
     static float sqrt_controller(float error, float p, float second_ord_lim);
 
+    // Inverse proportional controller with piecewise sqrt sections to constrain second derivative
+    static float stopping_point(float first_ord_mag, float p, float second_ord_lim);
```

Comment:
We will use this in higher level controllers. We currently have this code repeated in multiple places. it will probably all be moved to a nav_math library or something in the future.